ide/vscode/README.md

```diff
@@ -0,0 +1,11 @@
+# Sample configuration files for VSCode
+
+This example provides sample debug configurations that my be useful for developers using  the Microsoft(R) Visual Studio Code integrated development environment.
+
+A basic setup for Visual Studio Code is described in [Getting Started with the Raspberry Pi Pico](https://rptl.io/pico-get-started).
+
+To use the files copy them into a `.vscode` folder at the top level of your project and adapt them to your needs (see the [Visual Studio documentation](https://code.visualstudio.com/docs/cpp/launch-json-reference)).
```

Comment:
Sorry to nitpick, but is this "Visual Studio documentation" or "Visual Studio Code documentation" ?

Reply:
No problem - not 'nitpicking' just effective peer review ;-) Fixed in [d5cb160](https://github.com/raspberrypi/pico-examples/pull/313/commits/d5cb160cf07741a4538f696f4161b4dae0d600a9)